lib/middleware.ts

```diff
@@ -86,6 +103,14 @@ var defaultHandlers = [
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  () => ({source: "kayvee"}),
```

Comment:
this the `source` being `kayee` here is redundant information. "All" Kayvee logs are generated in this library. Source really should be the name of the application generation the log imo. This also makes it easier for us to do log searching and dashboards since we can always depend on `source`